### ISSUE TITLE
Incorrect creation_time

### ISSUE THREAD

**Issue by lim-james** [lim-james/wall-server#26]
Current Result
```
{
    "comment_id": 9,
    "post_id": 1,
    "user_id": 7,
    "text": "Hi I am commenting",
    "creation_time": "0001-01-01T00:00:00Z",
    "is_edited": false,
    "last_edited_time": "0001-01-01T00:00:00Z"
}
```

Possible fix - Just return comment_id
```
{
    "comment_id": 9
}
```

**Comment by lim-james** [comment]
Relates to #24

**Comment by lim-james** [comment]
Resolved in [ee273f0](https://github.com/lim-james/wall-server/commit/ee273f09907117aaa05d773ab3f42227662279ed)
